packages/flame/scripts/docgen.js

```diff
@@ -37,6 +39,70 @@ const getStyledSystemDocsUrl = parentPropName => {
   }
 };
 
+const namespaceType = parentName => {
+  switch (parentName) {
+    case 'WidthProps':
+    case 'HeightProps':
+    case 'MinWidthProps':
+    case 'MinHeightProps':
+    case 'MaxWidthProps':
+    case 'MaxHeightProps':
+    case 'DisplayProps':
+    case 'VerticalAlignProps':
+    case 'SizeProps':
+      return 'LayoutProps';
+    case 'FontFamilyProps':
+    case 'FontSizeProps':
+    case 'FontWeightProps':
+    case 'LineHeightProps':
+    case 'LetterSpacingProps':
+    case 'FontStyleProps':
+    case 'TextAlignProps':
+      return 'TypographyProps';
+    case 'AlignItemsProps':
+    case 'AlignContentProps':
+    case 'JustifyItemsProps':
+    case 'JustifyContentProps':
+    case 'FlexWrapProps':
+    case 'FlexDirectionProps':
+    case 'FlexProps':
+    case 'FlexGrowProps':
+    case 'FlexShrinkProps':
+    case 'FlexBasisProps':
+    case 'JustifySelfProps':
+    case 'AlignSelfProps':
+    case 'OrderProps':
+      return 'FlexboxProps';
+    case 'ZIndexProp':
+    case 'TopProps':
+    case 'RightProps':
+    case 'BottomProps':
+    case 'LeftProps':
+      return 'PositionProps';
+    case 'BorderWidthProps':
+    case 'BorderStyleProps':
+    case 'BorderColorProps':
+    case 'BorderRadiusProps':
+    case 'BorderTopProps':
+    case 'BorderRightProps':
+    case 'BorderBottomProps':
+    case 'BorderLeftProps':
+      return 'BorderProps';
+    case 'borderRadius':
+    case 'borderTopLeftRadius':
+    case 'borderTopRightRadius':
+    case 'borderBottomLeftRadius':
+    case 'borderBottomRightRadius':
+      return 'BorderRadiusProps';
+    case 'TextColorProps':
+    case 'BackgroundColorProps':
+    case 'OpacityProps':
+      return 'ColorProps';
+    default:
+      return parentName;
+  }
+};
+
```

Comment:
No way we can automate this list?

Reply:
This will require us to parse `styled-system`'s definition file and automagically pull everything out.